reposilite-backend/src/test/java/org/panda_lang/reposilite/repository/LookupControllerTest.java

```diff
@@ -97,11 +97,11 @@ void shouldReturn404WithUnauthorizedMessage() throws IOException {
 
     @Test
     void shouldReturn200AndProxiedFile() throws Exception {
-        super.reposilite.getConfiguration().setProxied(Collections.singletonList("http://localhost:8080"));
+        String proxyPort = String.valueOf(Integer.parseInt(testPort)+1);
```

Comment:
Vulnerability: In case of drawing 65535 as base number, it will fail trying to bi d 65536